### PR TITLE
add transform function that skips converting to fhir resources

### LINKED ISSUE #30: Add function to perform just the transformation
Currently the converter only works when converting the contents of the CSV to FHIR resources. The dataframe transformation/normalization logic is very powerful and useful on its own. We would benefit from utilizing this transformation logic to convert to other arbitrary data models.

### PR DESCRIPTION
**Pull request by hammadk373**
fixes #30 

Signed-off-by: Hammad Khan <480812+hammadk373@users.noreply.github.com>

### CODE REVIEW

File: tests/test_converter.py [hunk 1]
```diff
@@ -132,6 +132,29 @@ def patient_resource() -> Patient:
...
+        'familyName': 'Jones',
+        'ssn': '123-45-6789',
+        'state': 'MN',
+        'rowNum': 1,
+        'groupByKey': 'MRN1234',
+        'timeZone': 'US/Eastern',
+        'tenantId': 'sample-tenant',
+        'streamType': 'live',
+        'emptyFieldValues': ['empty', '\\n'],
+        'filePath': '/Users/hammadkhan/Projects/LinuxForHealth/CsvToFHIR/tests/resources/csv/2022-02-18-patient-fwf.dat',
```
Comment (cragun47) [inline tests/test_converter.py]: Hardcoded personal path should be removed.

**Review by cragun47 — CHANGES_REQUESTED** [review]
Remove hard-coded personal path.

Is this change backwards compatible with previous data-contracts?

File: tests/test_converter.py (hunk 1 above)
Comment (hammadk373) [reply tests/test_converter.py]: This change is completely backwards compatible, in fact the original behavior is still the default.

**Review by cragun47 — APPROVED** [review]
Thanks for changes

**Review by LisaWellman — APPROVED** [review]

**Pull request merged**